app/views/runs/_title.slim

```diff
@@ -1,50 +1,68 @@
-h3
-  = run.to_s
-- if run.completed?(Run::REAL) && run.completed?(Run::GAME)
-  = render partial: 'runs/timing_switch', locals: {timing: timing}
-h5.mt-3
-  div = format_milliseconds(run.duration_ms(timing))
+h1 = run.to_s
+h5
   - if run.user.try(:name)
-    small
-      span by #{link_to run.user, user_path(run.user), class: 'stealth-link user-link'}
-      - if run.user.silver_patron?
-        .gold-split title='Patron'
-h6
+    span> by
+    span = user_badge(run.user)
+    .badge
+      i.fas.fa-check>
+      span Personal Best
+  = render partial: 'runs/timing_badge', locals: {run: run, timing: timing}
 
-  ul.run-options
+ul.run-options
+  li
+    button#download-link.btn.btn-outline-success.dropdown-toggle href='#' role='button' data-toggle='dropdown' aria-haspopup='true' aria-expanded='false'
+      i.fas.fa-download>
+      span Download
+    .dropdown-menu.bg-dark aria-labelledby='download-link'
+      - timer = Run.program(run.timer)
+      a.dropdown-item.text-secondary href=download_path(run, timer.to_sym) = timer
+      a.dropdown-item.text-secondary href="https://one.livesplit.org/#/splits-io/#{run.id36}" target='_blank' LiveSplit One
+      - (Run.exportable_programs - [Run.program(run.timer)]).each do |timer|
+        a.dropdown-item.text-secondary href=download_path(run, timer.to_sym) = timer
+    - if run.category
```

Comment:
This will show up if its the only run on the site, which then gives you a link to compare it to itself.  I feel like this should just not be there in this case.

This also shows up if it's the best run on site, leading to the same behavior.